src/transport/SessionHandle.cpp

```diff
@@ -21,8 +21,25 @@
 
 namespace chip {
 
+using namespace Access;
 using namespace Transport;
 
+SubjectDescriptor SessionHandle::GetSubjectDescriptor() const
+{
+    SubjectDescriptor subjectDescriptor = { .fabricIndex = mFabric };
+    if (IsSecure())
+    {
+        subjectDescriptor.authMode    = AuthMode::kCase;
+        subjectDescriptor.subjects[0] = mPeerNodeId;
```

Comment:
Do we really want to introduce code that will erroneously claim that PASE communication (and possibly groupcast as well) is CASE?  This is effectively saying that mPeerNodeId is authenticated, when it might not be.

I realize we need to work incrementally toward large features.  But this is slightly concerning.

Reply:
PASE hook up to access control isn't supported yet. The work is tracked in issue #10242 and there's another issue #10245 to audit all the SubjectDescriptor hookup for correctness after it's done. The practical issue is we need to be able to run even incomplete code while we develop it. I've been running this code locally, but there's inordinate overhead keeping up with master, and if we don't put SubjectDescriptor in, other developers cannot add features to it or hook up the code that feeds it.

Reply:
Makes sense, and having #10245 in place seems sufficient.

Thanks.